web/src/js/views/read.tsx

```diff
@@ -136,6 +118,21 @@ export default class Read extends Page<Props, State> {
         this.initBook(book)
     }
 
+    @autobind
+    private async clickPage(e: MouseEvent) {
+        if (e.target instanceof HTMLElement) {
+            const per = e.x / e.target.getBoundingClientRect().width
+
+            if (per < 1 / 3) {
+                this.stepPage(-1)()
```

Comment:
We can change `stepPage` to just be a normal method instead of one that returns a function to call now.

It was like that before so I could stick calls to it within the onClick attribute, but with this change that's no longer done.